srcdeps-core-config-yaml/src/test/java/org/srcdeps/config/yaml/YamlConfigurationIoTest.java

```diff
@@ -36,7 +36,7 @@
  */
```

Comment:
The `YamlConfigurationIo.read()` was changed to return `Builder` that can be further customized rather than the immutable `Configuration`. The changes here in `YamlConfigurationIoTest` are just an adaptation to that return type change.